google/cloud/storage/client.h

```diff
@@ -76,47 +76,34 @@ struct ClientImplDetails;
  *
  * @par Credentials
  * The default approach for creating a Client uses Google Application Default
- * %Credentials (ADCs). Because finding or loading ADCs can fail, the returned
- * `StatusOr<Client>` from `CreateDefaultClient()` should be verified before
- * using it. However, explicitly passing `Credentials` when creating a Client
- * does not have the same potential to fail, so the resulting `Client` is not
- * wrapped in a `StatusOr`.  If you wish to use `AnonymousCredentials` or to
- * supply a specific `Credentials` type, you can use the functions declared in
- * google_credentials.h:
- * @code
- * namespace gcs = google::cloud::storage;
+ * %Credentials (ADCs). Note that a default-constructed client uses the ADCs:
```

Comment:
Is the '%' meaningful?

Reply:
Yes, `%Word` forces Doxygen to link `Word` to a class/struct/function with that name.  In any case, I no longer want the link, so I removed it.